assets/js/components/BatchEdit/About/Confirmation.jsx

```diff
@@ -140,28 +140,37 @@ const BatchEditConfirmation = ({
               </div>
             </section>
           )}
-
-          <div className="columns">
-            <div className="column is-three-fifths is-offset-one-fifth">
-              <div className="notification is-white">
-                <p className="has-text-danger has-text-centered mb-3">
-                  <FontAwesomeIcon icon="exclamation-triangle" /> NOTE: This
-                  will affect all works currently selected. Please proceed with
-                  extreme caution.{" "}
-                  <strong>To execute this change, type "I understand"</strong>
-                </p>
-
-                <UIFormInput
-                  errors={confirmationError}
-                  onChange={handleConfirmationChange}
-                  name="confirmationText"
-                  label="Confirmation Text"
-                  required
-                  data-testid="input-confirmation-text"
-                />
+          {!hasDeletes && !hasAdds ? (
+            <div className="notification is-white">
+              <p className="has-text-danger has-text-centered mb-3">
+                <FontAwesomeIcon icon="exclamation-triangle" />
+                <strong> NOTE: </strong> Please edit at least one variable to
+                submit Batch Edit Changes.
```

Comment:
@katdivyareddy10 Can we update this text to read "Batch edit currently only supports the following form items (Contributor, next time, etc...)".   And can we make the text not the red "danger" color and wrap in a `<p className="notification">...</p>`?

Reply:
I made changes, but since `<ul>` cannot be descendant of `<p>`, I moved it to outside of p and wrapped everything in a  `<div className="notification">`

Reply:
Looks great.